tests/integration/helpers/cluster.py

```diff
@@ -760,7 +760,7 @@ def add_instance(self, name, base_config_dir=None, main_configs=None, user_confi
                      hostname=None, env_variables=None, image="clickhouse/integration-test", tag=None,
                      stay_alive=False, ipv4_address=None, ipv6_address=None, with_installed_binary=False, tmpfs=None,
                      zookeeper_docker_compose_path=None, minio_certs_dir=None, use_keeper=True,
-                     main_config_name="config.xml", users_config_name="users.xml", copy_common_configs=True):
+                     main_config_name="config.xml", users_config_name="users.xml", copy_common_configs=True, config_root_name="yandex"):
```

Comment:
Why it's needed?

Reply:
Because YAML configs doesn't have root tag. We just hardcoded "clickhouse" in YAMLParser https://github.com/ClickHouse/ClickHouse/blob/dbff8fd03bd2ef52f147d0ca3f68626598820455/src/Common/Config/YAMLParser.cpp#L159. In integration tests we have several "predefined" configs, like https://github.com/ClickHouse/ClickHouse/blob/dbff8fd03bd2ef52f147d0ca3f68626598820455/tests/integration/helpers/0_common_instance_config.xml#L1-L20. In this configs we have `<yandex>` root tag. So when we try to run any test with YAML config we will be unable to start clickhouse (root tag mismatch). This parameter allows to change root tag in these "predefined" configs.